MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -121,7 +121,7 @@ class RouteManeuverViewController: UIViewController {
         
         if routeProgress.currentLegProgress.alertUserLevel == .arrive {
             distance = nil
-            destinationLabel.unabridgedText = routeStepFormatter.string(for: routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep))
+            destinationLabel.unabridgedText = routeProgress.remainingWaypoints.last?.name ?? routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep)
```

Comment:
We also need to repeat this line in `updateStreetNameForStep()`, somewhere before [this case](https://github.com/mapbox/mapbox-navigation-ios/blob/ff6f9e56ac213991e517d7eb3c13f3733bc8b558/MapboxNavigation/RouteManeuverViewController.swift#L175) (since the name is sometimes filled in by the Directions API). Otherwise, the name provided by the Directions API appears the entire time the user is on the destination’s street, and the name provided by the application appears only when nearing the destination itself.